client/state/reducer.js

```diff
@@ -94,4 +95,10 @@ const reducers = {
 	users,
 };
 
+// TODO: remove this once admin-menu state is consumed by UI components
```

Comment:
You can safely removed this entire block in this PR once you've rebased from `master` as the reducer will be async loaded on demand when you import the selectors or actions for use in the UI components.

Reply:
So I tried removing this, but then I was no longer able to get the state.  Is it because of my method of initially requesting it?
` 			reduxDispatch( requestAdminMenu( selectedSiteId ) );  `

Reply:
Was able to fix here: [add/sidebar-unified 240e795d53] Only init the admin-menu reducer when needed